Atomic/mount.py

```diff
@@ -824,6 +827,8 @@ def mount(self, identifier, options=None): # pylint: disable=arguments-differ
         if not OSTREE_PRESENT:
             return False
 
+        identifier = util.remove_skopeo_prefixes(identifier)
```

Comment:
the Docker backend doesn't use the "http:" prefix notation since insecure registries are configured in the configuration for the Docker daemon (i.e. `--insecure-registries`)

Reply:
> the Docker backend doesn't use the "http:" prefix notation 

I think I get it now =). So we will be assuming when using docker as storage, `atomic pull --storage=docker $image`, we do not need to worry about images starting with `ostree prefix` (http: or oci: or https:) right?

Reply:
yes correct, there won't be such prefixes for Docker images